src/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import marked from "marked";
+import "./index.css";
+
+class MarkdownViewer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      markdown : ''
+    }
+  }
+  handleChange(event) {
+    this.setState({markdown: marked.parse(event.target.value)})
+  }
+
+  render() {
+    const { markdown } = this.state;
+    return (
+      <div>
+        <textarea rows ="20" cols ="50" className ="data" placeholder="Enter the text" onChange={this.handleChange.bind(this)}></textarea> 
```

Comment:
this.handleChange.bind(this) when you put it in render, you are binding everytime the component is rendered. You could do this in constructor and avoid binding it everytime.
this.handleChange = e => this.handleChange(e)

Reply:
OK will try it.